lib/environments/meteorEnvironment.js

```diff
@@ -308,7 +300,8 @@ function meteorPackageNamedExports({ config }: Object): Object {
   // Try to identify the exports of all packages identified by
   // meteorPackageDependencies
   const namedExports = {};
-  for (const meteorPkg of meteorPackageDependencies({ config })) {
+
+  meteorPackageDependencies({ config }).forEach((meteorPkg: string) => {
```

Comment:
This change was supposed to go in the previous commit, but I accidentally put it here. FYI.